tests/Models/SqlUser.php

```diff
@@ -51,5 +57,12 @@ public static function executeSchema(): void
             $table->string('name');
             $table->timestamps();
         });
+        if (! $schema->hasTable('skill_sql_user')) {
+            $schema->create('skill_sql_user', function (Blueprint $table) {
```

Comment:
Do we really want to have a pivot table here? If the relations are already stored in the MongoDB document, it should not be duplicated in an SQL table.

Reply:
https://github.com/mongodb/laravel-mongodb/blob/fc1f9cc0b96d5d7d25fcb0eff7306d87c1a1efbc/src/Eloquent/HybridRelations.php#L290-L301

The sql models don't reach our `BelongsToMany` class. So, we can't control this behavior.
As I know, the sql models, store their relations data in a pivot tabel and mongo models, store theirs in a pivot column.